docs/02-app/01-building-your-application/04-caching/index.mdx

```diff
@@ -550,7 +550,7 @@ export async function generateStaticParams() {
 
 To statically render a subset of paths at build time, and the rest the first time they're visited at runtime, return a partial list of paths:
 
-```tsx filename="app/blog/[slug]/page.tsx" switcher
```

Comment:
if we aren't using `switcher` anymore for different file types I think we can delete this whole block